test/integration/suites/spire-server-cli/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM alpine:3.17 AS spire-server-alpine
+RUN apk add --no-cache --update openssl
+COPY --from=spire-server-scratch:latest-local /opt/spire/bin/spire-server /opt/spire/bin/spire-server
+ENTRYPOINT ["/opt/spire/bin/spire-server", "run"]
```

Comment:
Do the entrypoint first, this will allow better reuse of layer caching.

Reply:
Thanks for the suggestion. In this case, you would only notice a caching benefit if you ran this integration test multiple times successively with different scratch server images, which doesn't happen on CI. I agree it's an optimization, but it has pretty minor impact overall, so I think we will merge this as is for now, but we'd be happy to accept another PR with that change.